projects/pt1/python/torch_mlir/jit_ir_importer/build_tools/abstract_interp_lib_gen.py

```diff
@@ -5559,7 +5559,43 @@ def aten〇_make_per_tensor_quantized_tensor〡dtype(self_rank_dtype: Tuple[int,
       return torch.qint8
     return torch.qint32
 
+@check_shape_function([
+    Invocation(TensorOfShape(6, 4), 0, 2, 1), # Basic case.
```

Comment:
Maybe add a check for rank 0?

Reply:
Added case for rank = 0 with size = 1 and rank = 0 with size = 0